### DISCUSSION TITLE
how use multiple routes be like api rest

### DISCUSSION THREAD

**Discussion by LuWroblewski** [cargo-lambda/cargo-lambda#623] [Q&A] [answered]
I managed to run and deploy it but only / works, I tried using multiple routes and I'm not succeeding.

when trying to run localhost:9000/lambda-url/main or localhost:9000/lambda-url/notification in watch is succeful but in deploy not

several functions are created with different url, I'm trying to do

cargo lambda deploy main and cargo lambda deploy notification

I can't route with multiple routes? Only functions?

I tried to use cargo exus together but I couldn't

the folder organization is like this

src/bin
   - main.rs
   - notification.rs
   
Cargo.toml

**Comment by calavera** [comment, marked as answer]
You can do routing in one single function with Axum. There are multiple examples in the Runtime repository, search for "Axum": https://github.com/awslabs/aws-lambda-rust-runtime/tree/main/examples

**Reply by LuWroblewski** [reply]
Thank you very much, I've been try for hours, everything works fine now :)
